.changeset/five-sloths-hang.md

```
@@ -0,0 +1,8 @@
+---
+"@vahor/typed-es": patch
+---
+
+Add support for generic function in aggregations.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)